docs/installation/config.rst

```diff
@@ -376,6 +366,33 @@ Other settings
 
 .. _`Django DATABASE settings`: https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-DATABASE-ENGINE
 
+.. _installation_environment_config_feature_flags:
+
+Feature flags
+=============
+
+Open Forms sometimes supports a layered approach for feature flags, where some
+behaviours can be enabled at deploy-time through environment variables already. If
+this option is not available, you can still enable/disable the feature flag in the
+admin interface, via **Admin** > **Configuration** > **Flag states**.
```

Comment:
For my understanding: does setting the feature flag via the admin always override the environment variable behavior?

Reply:
depends on the `required` key being set or not - if you define it as `required` in code and the envvar disables it, then whatever you do in the admin will not matter.